localstack/services/s3/constants.py

```diff
@@ -66,3 +66,5 @@
     "X-Amz-SignedHeaders",
     "X-Amz-Signature",
 ]
+
+S3_CHUNK_SIZE = 1024 * 16 * 4
```

Comment:
What exactly is this constant here? It's 64k, but why is that? Could you please add a comment?

Reply:
I'll add a comment, this was the result of quick tests with the S3 streaming to see what kind of value would have impact on the memory usage vs time spent iterating over the stream. Not very empirically chosen I have to say.